app/components/settings/categories/GeneralSettings.js

```diff
@@ -10,13 +10,29 @@ import LocalizableError from '../../../i18n/LocalizableError';
 import styles from './GeneralSettings.scss';
 import type { ReactIntlMessage } from '../../../types/i18nTypes';
 import FlagLabel from '../../widgets/FlagLabel';
+import { tier1Languages } from '../../../config/languagesConfig';
 
 const messages = defineMessages({
   languageSelectLabel: {
     id: 'settings.general.languageSelect.label',
     defaultMessage: '!!!Language',
   },
-
+  languageSelectLabelInfo: {
```

Comment:
Maybe better to make these global messages instead of duplicating them? (see `global-messages.js`)

You could even add a helper method in there that constructs the `contributors` string so you don't have to duplicate this logic either